tests/online_ddl/run.sh

```diff
@@ -43,58 +46,67 @@ function real_run() {
 		"\"result\": true" 1
 
 	# imitate a DM task is started during the running of online DDL tool
-	if [ "$online_ddl_scheme" == "gh-ost" ]; then
-		run_sql_source1 "create table online_ddl.ignore (c int); create table online_ddl._ignore_gho (c int);"
-	elif [ "$online_ddl_scheme" == "pt" ]; then
-		run_sql_source1 "create table online_ddl.ignore (c int); create table online_ddl._ignore_new (c int);"
-	fi
+	run_sql_source1 "create table online_ddl.gho_ignore (c int); create table online_ddl._gho_ignore_gho (c int);"
+	run_sql_source1 "create table online_ddl.pt_ignore (c int); create table online_ddl._pt_ignore_new (c int);"
```

Comment:
Why did the former test not report this error

```log
{
    "result": false,
    "msg": "[code=26005:class=dm-master:scope=internal:level=medium], Message: fail to check synchronization configuration with type: check was failed, please see detail
        detail: {
                "results": [
                        {
                                "id": 7,
                                "name": "table structure compatibility check",
                                "desc": "check compatibility of table structure",
                                "state": "fail",
                                "errors": [
                                        {
                                                "severity": "fail",
                                                "short_error": "table `online_ddl`.`_gho_ignore_gho` primary/unique key does not exist",
                                                "instruction": "please set primary/unique key for the table"
                                        },
                                        {
                                                "severity": "fail",
                                                "short_error": "table `online_ddl`.`gho_ignore` primary/unique key does not exist",
                                                "instruction": "please set primary/unique key for the table"
                                        }
                                ],
                                "extra": "address of db instance - 127.0.0.1:3306"
                        }
                ],
                "summary": {
                        "passed": false,
                        "total": 18,
                        "successful": 17,
                        "failed": 1,
                        "warning": 0
                }
        }",
    "sources": [
    ]
}
```

Reply:
Done. The regular expression is wrong. https://github.com/pingcap/dm/pull/1904/files#diff-13484eff718862326f14f11f7fefeb5b0cc8d2688932e779de36e77c3da6d39bR42